apps/seasons-sunray-angle/seasons-sunray-angle.less

```diff
@@ -0,0 +1,44 @@
+.overlay {
+  &.grasp-seasons {
+    position: fixed;
+    background-color: transparent;
+    text-align: center;
+
+    &.main{
+      top: 10px;
+      left: 10px;
+      width: 795px;
+      height: 600px;
+    }
+    &.small-top{
+      top: 10px;
+      left: 815px;
+      width: 395px;
+      height: 300px;
+    }
+    &.small-bottom{
+      top: 340px;
+      left: 815px;
+      width: 395px;
+      height: 300px;
```

Comment:
Still can see text overflowing small-bottom container:
![screen shot 2016-08-12 at 11 04 04](https://cloud.githubusercontent.com/assets/767857/17619662/23d8e436-6087-11e6-8271-614772959c2c.png)
I'm betting it's just matter of some CSS tweaks here?